### ISSUE TITLE
Fileupload not respecting config values

### ISSUE THREAD

**Issue by ClausMunch** [livewire/livewire#1263]
### Description
When using a S3 compatible disk, Livewire Fileupload builds .amazonaws.com strings to upload files and thus not respecting the config in filesystems.php

### Steps to reproduce
Basic example of a clean Laravel install w. livewire and minio or digitalocean space configured.
Basic tutorial from the documentation will fail and try to upload file to an {bucket}.S3.{region}.amazonaws.com address.

**Context**
- Livewire version: dev-master ad2f1fd
- Laravel version: v7.19.1
- Browser: Chrome

**Comment by iAmKevinMcKee** [comment]
Are you sure you have the correct region? Can you double check that your .env has the right code from here: https://docs.aws.amazon.com/general/latest/gr/rande.html

**Comment by ClausMunch** [comment]
Yes, I'm sure. I'm not using an Amazon S3, but have tried with both digitalocean spaces and Minio S3 storage.
Works brilliantly, except for with Livewire.
